### ISSUE TITLE
mac .dmgs "damaged

### ISSUE THREAD

**Issue by diego-** [sharpie7/circuitjs1#935]

**Comment by diego-** [comment]
The referenced offline packages of CircuitJS are available here: http://www.falstad.com/circuit/offline/

The .dmg files generated for MacOS do not seem to be properly configured. When the disk image is opened, MacOS complains that it is damaged and must be ejected.

**Comment by pfalstad** [comment]
They work fine for me.  What version of the OS are you on?  I'm running Sonoma.

**Comment by diego-** [comment]
I'm running Sonoma 14.4.1 on an Apple M1 Max chip. 

Attached a screenshot of the issue for more detail.
![Screenshot 2024-04-24 at 10 20 55 AM](https://github.com/sharpie7/circuitjs1/assets/2730503/35998dda-1447-4820-acbb-1b45d9ec99b7)


(love the circuit simulator btw, great job!)

**Comment by pfalstad** [comment]
ok the app doesn't work for me actually, I thought you meant opening the dmg doesn't work.  Hmm.

**Comment by pfalstad** [comment]
If you do "xattr -cr CircuitJS1.app" it will run, but I'll try to figure out why you need to do that.

**Comment by diego-** [comment]
you're right, its an app issue. I tried running it from the opened DMG directly and got that message. When I drag it to Applications I get a slightly different version of the same error. 

xattr solution works, thanks!
